Packs/Phishing/Playbooks/Process_Email_-_Generic_v2.yml

```diff
@@ -1726,13 +1939,29 @@ inputs:
     v2".
   playbookInputQuery:
 - key: Thread-Topic
-  value: {}
+  value:
+    complex:
+      root: incident
+      accessor: emailsubject
   required: false
   description: The value of the "Thread-Topic" header which holds the original email
     subject. This is necessary for forwarded emails scenarios. It will be passed as
     an input to the "Get Original Email - Generic v2" playbook to be used in the relevant
     sub-playbooks.
   playbookInputQuery:
+- key: EmailBrand
+  value: {}
+  required: false
+  description: |-
+    When this value is provided, only the relevant sub-playbook of "Get Original Email - Generic v2" will be executed.
```

Comment:
need to change the description here as well

Reply:
Fixed